cmd/cosign/webhook/main.go

```diff
@@ -92,7 +92,7 @@ func NewValidatingAdmissionController(ctx context.Context, cmw configmap.Watcher
 
 	return validation.NewAdmissionController(ctx,
 		// Name of the resource webhook.
-		webhookName,
+		*webhookName,
```

Comment:
Note: If you change this, you will to change on the scripts too.

Reply:
Do you mean the helm [templates](https://github.com/sigstore/helm-charts/blob/main/charts/cosigned/templates/webhook/webhook_mutating.yaml) and [config files](https://github.com/sigstore/cosign/blob/main/config/500-webhook-configuration.yaml)? If yes, I rely on this comment https://github.com/sigstore/cosign/blob/56af7f3f51556cb2374d6556ad08e5a2b1d764a2/cmd/cosign/webhook/main.go#L47-L49 to get the attention to whomever tries to change it. Do you suggest to enhance the help string of the flag?

Reply:
Yes, we'll need to update the comment in that piece of code accordingly. 
Also we'll need to add that field in the values.yaml of the helm-chart, so everything gets changed based on the value of the field. I could take care of that.

Reply:
I have updated the comment and the flag doc with 62e602886e8101ae2768e1395662cc8de10e26ad. 
Looking in the webhook initialization in [1] and [2] there is a lot of configurations that can be injected, but also skipped in different conditions, or even fail when URL is used instead of in-cluster service.  I doubt I can describe all of them with just few words, and I don't think here is the right place for detailed documentation. 
If you have better suggestion/wording, please propose it.  

[1] https://github.com/knative/pkg/blob/e325df66cb51d341b5d1142fa42f75539dfe290a/webhook/resourcesemantics/validation/reconcile_config.go#L99 
[2] https://github.com/knative/pkg/blob/e325df66cb51d341b5d1142fa42f75539dfe290a/webhook/resourcesemantics/defaulting/defaulting.go#L170

Reply:
We'll get some documentation for cosigned new changes soon.